README.md

```diff
@@ -16,20 +16,21 @@ You can add Fluent Kafka Streams Tests via Maven Central.
 
 #### Gradle
 ```gradle
-compile group: 'com.bakdata', name: 'fluent-kafka-streams-tests', version: '1.1.0'
+compile group: 'com.bakdata', name: 'fluent-kafka-streams-tests-junit5', version: '2.0.0'
```

Comment:
The changes here all point to the junit5 version but the badges do not. This should be made consistent.